src/main/java/com/liferay/faces/util/product/internal/ProductBaseImpl.java

```diff
@@ -39,7 +39,7 @@
 	protected int patchVersion;
 	protected String stringValue;
 	protected String title;
-	protected String version;
+	protected String version = majorVersion + "." + minorVersion + "." + patchVersion;
```

Comment:
@stiemannkj1 I think this might result in a String with value "0.0.0"? This logic may have to move to the init method?